backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/theme/BpkTheme.kt

```diff
@@ -75,11 +76,23 @@ object BpkTheme {
 
   val typography: BpkTypography
     @Composable
-    get() = LocalBpkTypography.current
+    get() = if (LocalInspectionMode.current) {
```

Comment:
this is the actual change that makes it work. I originally wanted to make the change within the `LocalBpkTypography` definition, but that is not working as it's not a composable.
I tested the behaviour in preview & in the app and the right version is used in both situations.